### ISSUE TITLE
Style disappears after editing door

### ISSUE THREAD

**Issue by myoualid** [IfcOpenShell/IfcOpenShell#3593]
Scenario:
I create a door type with the type menu.
I assign a style to this door Type.
I set the door parameter "Overall Height" to 2.2 
I press "bim.finish_editing_door"

Oups: The Style disappears

( In the sceneario where I assign an ifc material and then assign a style, it's all good though)

**Comment by Moult** [comment]
Yeah I'd like to hold off from implementing this stuff until more groundwork is done on the new item-based style / material UI.

**Comment by Moult** [comment]
Closing as now parametric doors can have attached styles.
